quetz/cli.py

```diff
@@ -483,5 +485,30 @@ def list() -> NoReturn:
         typer.echo('\n'.join([p for p in deployments]))
 
 
+@app.command()
+def plugin(cmd: str, path: str = typer.Argument(None, help="Path to the plugin folder")) -> NoReturn:
+
+    if cmd == 'install':
+        abs_path = Path(path).absolute()
+        assert (abs_path / "setup.py").exists()
+
+        exes = ['micromamba', 'mamba', 'conda']
+        if (abs_path / "requirements.txt").exists():
+            for exe in exes:
+                exe_path = find_executable(exe)
+                if exe_path:
+                    break
+
+        if not exe_path:
```

Comment:
what if requirements.txt does not exist?

Reply:
handled that case now